### PR TITLE
Document how to add columns to Open Data

### PR DESCRIPTION
**Pull request by wrridgeway**

### CODE REVIEW

File: How-To/Add-columns-to-open-data-assets.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Add Columns to Open Data Assets
```
Comment (ccao-jardine) [inline How-To/Add-columns-to-open-data-assets.md:1]: Minor suggestion: change title to "Add Columns to an existing Open Data asset"

**Review by ccao-jardine — COMMENTED** [review]
**Praise:** Nice! Good job remembering the finer details like scheduling the update. One minor change suggested.

File: How-To/Add-columns-to-open-data-assets.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The data team maintains a motley collection of [open data assets](https://datacatalog.cookcountyil.gov/browse?tags=cook+county+assessor). The [Athena views](https://ccao-data.github.io/data-architecture/#!/overview/ccao_data_athena) that feed these assets are updated regularly with new features, and it's important to maintain parity.
+
+1. [Log in to](https://datacatalog.cookcountyil.gov/login) the open data portal using the Data team's credentials.
```
Comment (dfsnow) [inline How-To/Add-columns-to-open-data-assets.md:5]:
```suggestion
1. [Log in to](https://datacatalog.cookcountyil.gov/login) the Open Data Portal using the Data Department's credentials.
```

I _think_ this is correct? Technically Open Data Portal is a proper noun? Or maybe it's just a noun, idk.

File: How-To/Add-columns-to-open-data-assets.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+2. Click "My Assets" on the left.
```
Comment (dfsnow) [inline How-To/Add-columns-to-open-data-assets.md:6]: **nitpick:** We used backticks before for differentiating actions and fields. i.e.: 

```suggestion
2. Click `My Assets` on the left.
```

**Review by dfsnow — APPROVED** [review]
Tiny nitpicks but otherwise LGTM

**Pull request merged**